lib/polyfill/v2_6/matrix.rb

```diff
@@ -3,6 +3,21 @@
 module Polyfill
   module V2_6
     module Matrix
+      def []=(i, j, v)
+        raise FrozenError, "can't modify frozen Matrix" if frozen?
+        (rows = check_range(i, :row)) || (row = check_int(i, :row))
```

Comment:
@AaronLasseigne How should associated private methods be brought in by Polyfill?

Reply:
I haven't had to bring in private methods yet. There are basically two options that I can think of. Either we inline the private code (which is kind of annoying) or we have a way to associate the private methods and import them along with the method requested (which is a totally new feature).